src/components/core/DpUpload/utils/UppyTranslations.js

```diff
@@ -8,9 +8,9 @@ const de = () => {
        * Text to show on the droppable area.
        * `%{browse}` is replaced with a link that opens the system file selection dialog.
        */
-      dropHereOr: Translator.trans('form.button.upload.pdf', {
+      dropHereOr: german.upload.select.pdf({
         browse: '{browse}',
-        maxUploadSize: convertSize('GB', window.dplan.settings.maxUploadSize)
+        maxUploadSize: convertSize('GB', 2)
```

Comment:
Deviating from @hwiem 's suggestion, I would recommend to make this value a prop, because this is passed from the backend, where it is derived from server settings, so that each project may have its own, real max upload size. When using DpUploadFIles in a project whose server is configured with 256MB max_upload_size, the information would otherwise be plain wrong.

Reply:
When looking at usages of DpUploadFiles, we didn't find a single instance that didn't override this translation key (and the `maxUploadSize` along with it) via `translations` prop, so this default value is actually never used. If one wanted to pass the `maxUploadSize` as defined in the GlobalConfig to the translation key, one could already do so via the translations prop.

Reply:
I see... did not look closely enough. You're right.